src/model/index.ts

```diff
@@ -358,10 +390,7 @@ class Model implements WorkbookModel {
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private TX_PLAN_SECURITY(value: any) {
-    if (
-      value.object_type === "TX_PLAN_SECURITY_ISSUANCE" ||
```

Comment:
I've made this separation because contrary to the `TX_PLAN_SECURITY_ISSUANCE`, we rely on .stock_class_id for `TX_EQUITY_COMPENSATION_ISSUANCE` and not `.stock_plan_id`